tests/integration/static_ip_address_test.py

```diff
@@ -16,6 +16,7 @@
 #
 
 import copy
+import pytest
```

Comment:
Please separate from the above with a newline

Reply:
Done